2018-12-27 18:55:27,233 - logs/main.logs:15 - WARNING - logs/main.logs
2018-12-27 18:56:12,629 - logs/main.logs:15 - WARNING - logs/main.logs
2018-12-28 00:17:40,917 - logs/main.logs:15 - WARNING - logs/main.logs
2018-12-28 00:54:48,935 - logs/main.logs:15 - WARNING - logs/main.logs
2018-12-28 00:57:03,788 - logs/main.logs:15 - WARNING - logs/main.logs
2018-12-28 01:04:33,303 - logs/main.logs:15 - WARNING - logs/main.logs
2018-12-28 01:14:11,365 - logs/main.logs:15 - WARNING - logs/main.logs
2018-12-28 01:34:25,965 - logs/main.logs:15 - WARNING - logs/main.logs
2018-12-28 01:35:27,708 - logs/main.logs:15 - WARNING - logs/main.logs
2018-12-28 01:50:36,418 - logs/main.logs:15 - WARNING - logs/main.logs
2018-12-28 01:52:47,102 - logs/main.logs:15 - WARNING - logs/main.logs
2018-12-28 01:57:14,479 - logs/main.logs:15 - WARNING - logs/main.logs
2018-12-28 02:01:17,663 - logs/main.logs:15 - WARNING - logs/main.logs
2018-12-28 08:08:22,863 - logs/main.logs:15 - WARNING - logs/main.logs
2018-12-28 08:27:06,041 - logs/main.logs:15 - WARNING - logs/main.logs
2018-12-28 08:42:33,857 - logs/main.logs:15 - WARNING - logs/main.logs
2018-12-28 08:43:56,501 - logs/main.logs:15 - WARNING - logs/main.logs
2018-12-28 08:57:14,557 - logs/main.logs:15 - WARNING - logs/main.logs
2018-12-28 09:00:06,304 - logs/main.logs:15 - WARNING - logs/main.logs
2018-12-28 09:12:33,421 - logs/main.logs:15 - WARNING - logs/main.logs
2018-12-28 09:13:19,075 - logs/main.logs:33 - ERROR - TABS:
Traceback (most recent call last):
  File "main.py", line 28, in aion_analytics
    ch = yield churn_tab()
  File "/home/andre/anaconda3/envs/bokeh_aion_analytics/lib/python3.5/site-packages/tornado/gen.py", line 1133, in run
    value = future.result()
  File "/home/andre/anaconda3/envs/bokeh_aion_analytics/lib/python3.5/asyncio/futures.py", line 294, in result
    raise self._exception
  File "/home/andre/anaconda3/envs/bokeh_aion_analytics/lib/python3.5/site-packages/tornado/gen.py", line 307, in wrapper
    result = func(*args, **kwargs)
  File "/home/andre/anaconda3/envs/bokeh_aion_analytics/lib/python3.5/types.py", line 243, in wrapped
    coro = func(*args, **kwargs)
  File "/home/andre/aion/data_science/bokeh/aion_analytics/scripts/dashboards/churn.py", line 63, in churn_tab
    class Thistab(Mytab):
  File "/home/andre/aion/data_science/bokeh/aion_analytics/scripts/dashboards/churn.py", line 75, in Thistab
    ref_block_tx_warehouse_tab = Mytab('block_tx_warehouse', cols, [])
  File "/home/andre/aion/data_science/bokeh/aion_analytics/scripts/utils/mytab.py", line 34, in __init__
    self.df = streaming_dataframe.get_df()
  File "/home/andre/aion/data_science/bokeh/aion_analytics/scripts/streaming/streamingDataframe.py", line 64, in get_df
    return self.df
AttributeError: 'StreamingDataframe' object has no attribute 'df'
2018-12-28 09:16:33,918 - logs/main.logs:15 - WARNING - logs/main.logs
2018-12-28 09:32:44,352 - logs/main.logs:15 - WARNING - logs/main.logs
2018-12-28 09:43:45,435 - logs/main.logs:15 - WARNING - logs/main.logs
2018-12-28 09:49:49,391 - logs/main.logs:15 - WARNING - logs/main.logs
2018-12-28 09:52:17,367 - logs/main.logs:15 - WARNING - logs/main.logs
2018-12-28 09:55:43,673 - logs/main.logs:15 - WARNING - logs/main.logs
2018-12-28 09:57:01,260 - logs/main.logs:15 - WARNING - logs/main.logs
2018-12-28 10:02:47,174 - logs/main.logs:15 - WARNING - logs/main.logs
2018-12-28 10:11:43,342 - logs/main.logs:15 - WARNING - logs/main.logs
2018-12-28 10:15:16,859 - logs/main.logs:15 - WARNING - logs/main.logs
2018-12-28 10:17:30,899 - logs/main.logs:15 - WARNING - logs/main.logs
2018-12-28 10:24:27,364 - logs/main.logs:15 - WARNING - logs/main.logs
2018-12-28 10:30:23,174 - logs/main.logs:15 - WARNING - logs/main.logs
2018-12-28 10:33:44,308 - logs/main.logs:15 - WARNING - logs/main.logs
2018-12-28 10:35:22,904 - logs/main.logs:15 - WARNING - logs/main.logs
2018-12-28 10:38:57,688 - logs/main.logs:15 - WARNING - logs/main.logs
2018-12-28 10:42:03,867 - logs/main.logs:15 - WARNING - logs/main.logs
2018-12-28 10:51:42,895 - logs/main.logs:15 - WARNING - logs/main.logs
2018-12-28 10:54:22,606 - logs/main.logs:15 - WARNING - logs/main.logs
2018-12-28 11:07:50,560 - logs/main.logs:15 - WARNING - logs/main.logs
2018-12-28 11:17:03,391 - logs/main.logs:15 - WARNING - logs/main.logs
